src/pages/ComicBreakdownPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Divider, Button } from "@nextui-org/react";
+import { IComicData } from "../models/comicData";
+import Navigation from "../elements/Navigation";
+
+function ComicBreakdownPage({ comic }: { comic: IComicData }) {
+  return (
+    <>
+      <Navigation isLoggedIn={true}></Navigation>
+      <div className="max-w-md">
+        <div className="space-y-1">
+          <h4 className="text-medium font-medium">{comic.name}</h4>
```

Comment:
As far as I know, `text-medium` is not a valid Tailwind class, you could use `text-base` for a font size of 16px, or `text-lg` for a font size of 18px.

Reply:
Yes, it is another nextUI tailwind class 

```
.text-medium {
    font-size: var(--nextui-font-size-medium);
    line-height: var(--nextui-line-height-medium);
}
```